sdjournal/read.go

```diff
@@ -185,6 +185,10 @@ func (r *JournalReader) Follow(until <-chan time.Time, writer io.Writer) (err er
 	// Process journal entries and events. Entries are flushed until the tail or
 	// timeout is reached, and then we wait for new events or the timeout.
 	var msg = make([]byte, 64*1<<(10))
+	var waitCh = make(chan int)
+	var waitStop = make(chan bool)
```

Comment:
If I'm not mistaken, these used to be 1-slot buffered channels. I think the change is on purpose to synchronize sender and receiver, right? Are there other hidden behavioral changes behind this?